src/receiver/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = class Receiver extends EventEmitter {
 
     this.verify_token = opts.verify_token
     this.context = opts.context
+    this.log = opts.log
```

Comment:
I think we've tried to consolidate where we log up a level where we have a logger object and handle formatting consistently. If we change this so the verify token module emits an error event that this receiver could bubble up then the main slapp module could handle listening to receiver errors and logging them like other things are logged.

It does mean the verify token module gets slightly more complicated, but not much.